Podverse/MakeClipTimeViewController.swift

```diff
@@ -113,27 +117,62 @@ class MakeClipTimeViewController: UIViewController, UITextFieldDelegate {
         }
         
         updateSpeedLabel()
+        loadMakeClipInputs()
+
+        if self.editingItem == nil {
+            self.title = "Make Clip"
+        } else {
+            self.title = "Edit Clip"
+        }
         
+        if (!hasSeenHint) {
+            animateHelper()
+        } else {
+            self.hintView.removeFromSuperview()
+            setupBarButtonItems()
+        }
+    }
+    
+    func animateHelper() {
+        if(self.shouldAnimate) {
+            if self.hintImageHorizontalConstraint != nil {
+                self.hintImageHorizontalConstraint.constant += 200
```

Comment:
Right now it looks like we're using fixed size constants for the constraints, but won't that mean on some devices the swiping will happen off-center?

I also think it would look a little nicer if the animation could start in the middle, then move right 100, then left 200, then right 200, etc. rather than the full width of the rectangle. Do you have an idea of how that could work?

Reply:
Ah I see now. Makes sense. Alright between this and the other issue, a gif seems worth it. I'll merge this and see if I can make the gif myself.